addons/sourcemod/scripting/soap_tf2dm.sp

```diff
@@ -356,22 +354,70 @@ void InitSpawnSys()
         }
         else
         {
-            SetFailState("Map spawns missing. Map: %s, no cURL support", map);
+            LogMessage("Map spawns missing. Map: %s, no cURL support", map);
             LogError("File Not Found: %s, no cURL support", path);
+            SetDefaultSpawns(true, true);
         }
     }
     // End spawn system.
 }
 
-// lmfao dude kill me
-public LoadMapConfig(const char[] map, const char[] path)
+public SetDefaultSpawns(bool setRed, bool setBlu)
```

Comment:
this can just be `void SetDefaultSpawns(bool setRed, bool setBlu)`, no need to make it public